### ISSUE TITLE
invalid `.github/workflow` folder

### ISSUE THREAD

**Issue by D4nte** [acid-info/waku.org#3]
There is a `.github/workflow` folder with a `.yml` file. Pretty sure these aren't use by GitHub. Not sure if it comes from acid-info builder.

**Comment by amirhouieh** [comment]
Thatis right! it was skipped through from the begining. In my list for post launch to remove it.
